core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -63,20 +63,34 @@ private[spark] class CoarseMesosSchedulerBackend(
   // Maximum number of cores to acquire (TODO: we'll need more flexible controls here)
   val maxCores = conf.get("spark.cores.max",  Int.MaxValue.toString).toInt
 
+  val maxExecutorsPerSlave =
+    conf.getOption("spark.mesos.coarse.executors.max").map { m => m.toInt }
+
+  val maxCpusPerExecutor =
+    conf.getOption("spark.mesos.coarse.coresPerExecutor.max").map { m => m.toInt }
```

Comment:
I prefer `spark.mesos.coarse.maxCoresPerExecutor`, since we don't really expect anything else to fall under the namespace `spark.mesos.coarse.coresPerExecutor.*`. I wouldn't follow the naming for `spark.cores.max` if possible :)